src/css/profile/mobile/common/button.less

```diff
@@ -287,12 +287,6 @@ a.ui-btn {
 			width: 208 * @unit_base;
 		}
 
-		&::after {
-			width: 60 * @unit_base;
-			height: 60 * @unit_base;
-			background-color: var(--button-icon-color);
-		}
-
```

Comment:
is it neccessary part for the button?
It make some weird result in fab button.
![image](https://user-images.githubusercontent.com/29908741/79825337-b8346100-83d3-11ea-91a2-3c4a16a4ae85.png)